SparkPost/2web2ui
# FE-1299 - Fixes layout problem with the Pagination component

[FE-1299](https://sparkpost.atlassian.net/browse/FE-1299)

### What Changed

- Uses newer Matchbox `Columns` component to fix layout issues with the `TableCollection` component on smaller screen sizes
- On smaller screen sizes the layout looks like this:
<img width="659" alt="Screen Shot 2021-01-22 at 9 16 48 AM" src="https://user-images.githubusercontent.com/3613392/105501798-a64d5a80-5c92-11eb-8138-306f28fafaea.png">
- On larger screen sizes the layout looks like this:
<img width="1320" alt="Screen Shot 2021-01-22 at 9 16 39 AM" src="https://user-images.githubusercontent.com/3613392/105501817-aea59580-5c92-11eb-9513-5c16f2a9689a.png">


### How To Test

- Visit a page using the `TableCollection` (like **/lists/recipient-lists** or **/domains**) and resize the window until it is relatively narrow
- Verify that pagination content wraps as expected
- Verify that the pagination buttons conditionally render as expected according to the changes

### To Do

- [X] Wait on UX feedback: https://msys.slack.com/archives/GLLAW6UAX/p1611261474009100
